docs/layouts/shortcodes/generated/kubernetes_operator_config_configuration.html

```diff
@@ -80,6 +80,24 @@
             <td>Boolean</td>
             <td>Whether to enable recovery of missing/deleted jobmanager deployments.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.job.health-check.enabled</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to enable health check for jobs.</td>
+        </tr>
+        <tr>
+            <td><h5>kubernetes.operator.job.restart-check.duration-window</h5></td>
+            <td style="word-wrap: break-word;">2 min</td>
+            <td>Duration</td>
+            <td>The duration of the time window where job restart count measured.</td>
+        </tr>
+        <tr>
+            <td><h5>kubernetes.operator.job.restart-check.threshold</h5></td>
```

Comment:
There is some inconsistency in the naming here, `health-check` vs `restart-check`. 
Maybe it could be:
```
kubernetes.operator.job.health-check.enabled
kubernetes.operator.job.health-check.window
kubernetes.operator.job.health-check.restarts.threshold / kubernetes.operator.job.health-check.num- restarts.threshold

```

Reply:
Good point, renamed to:
```
kubernetes.operator.job.health-check.enabled
kubernetes.operator.job.health-check.restarts.window
kubernetes.operator.job.health-check.restarts.threshold
```